### ISSUE TITLE
swp files

### ISSUE THREAD

**Issue by maisano** [nickjj/manifest-revision-webpack-plugin#25]
thanks for the plugin :)

unfortunately, i've noticed that my vim swap files are being picked up by it:
```
ERROR in ./src/containers/.Login.js.swp
Module parse failed: ~/dev/project/src/containers/.Login.js.swp Unexpected token (1:6)
You may need an appropriate loader to handle this file type.
```

not sure it's _that_ big of a deal though–i fixed it by changing where my `.swp` files get saved.

**Comment by nickjj** [comment]
Hi,

Were you seeing this issue without using this plugin?

**Comment by maisano** [comment]
nope. the error would only occur with the plugin enabled.

**Comment by nickjj** [comment]
Let me know if PR #19 fixes things for you. It introduced a new option that you can set to white list files.

**Comment by maisano** [comment]
i should have mentioned that i still get this error as of `0.3.0`. that regex unfortunately is a whitelist and doesn't exclude other matches.

i should maybe have a moment to dig deeper into this tonight.

**Comment by nickjj** [comment]
Right, the PR alone wouldn't fix it. You would be responsible for supplying your own white list, in which case it would ignore the swp file since you wouldn't add it to the list.

**Comment by maisano** [comment]
not quite. it's one of two conditions that dictates whether a file gets included. it doesn't match against regex but the next condition picks it up.

**Comment by maisano** [comment]
looks like the issue is that `walkAndPrefetchAssets` is adding arbitrary files to the bundle.

**Comment by dleavitt** [comment]
@vladra @maisano 

I don't think the extensions regex has anything to do with this (it's for including files that would otherwise be excluded.)

Luckily ignorePaths accepts regexes, so you can get the results you want like this:
```javascript
ignorePaths: [
  /\.(|min\.js)$/,
  /^\./
]
```
